.circleci/config.yml

```diff
@@ -77,27 +77,28 @@ jobs:
               apply \
                 -auto-approve \
                 -var package_path=${DEBNAME} \
-                -var instance_name=circleci-terraform-${CIRCLE_SHA1}
+                -var instance_name=circleci-terraform-${CIRCLE_SHA1} \
+                -var additional_files_dir=${PWD}/scripts/ci/tests/
       - run:
-          name: Install and run deb
+          name: Install deb
           command: |
             set -x
-            ec2_ip=$(terraform -chdir=scripts/ci output -raw test_node_ssh)
+            export ec2_ip=$(terraform -chdir=scripts/ci output -raw test_node_ssh)
             ssh -o "StrictHostKeyChecking=no" ubuntu@$ec2_ip \<< EOF
             sudo apt-get update && sudo apt-get install -y /home/ubuntu/influxdb.deb
-            sudo service influxdb start
-            EOF
-            ssh -o "StrictHostKeyChecking=no" ubuntu@$ec2_ip \<< EOF
-            sudo service influxdb stop
-            sudo sed -i 's/  # https-enabled = false/  https-enabled = true/' /etc/influxdb/influxdb.conf
-            sudo sed -i 's|  # https-certificate = "/etc/ssl/influxdb.pem"|  https-certificate = "/etc/ssl/influxdb.crt"|' /etc/influxdb/influxdb.conf
-            sudo sed -i 's|  # https-private-key = ""|  https-private-key = "/etc/ssl/influxdb.key"|' /etc/influxdb/influxdb.conf
-            sudo openssl req -x509 -nodes -newkey rsa:2048 -keyout /etc/ssl/influxdb.key -out /etc/ssl/influxdb.crt -days 365 -subj /C=US/ST=CA/L=sanfrancisco/O=influxdata/OU=edgeteam/CN=localhost
-            sudo chown influxdb:influxdb /etc/ssl/influxdb.*
-            sudo service influxdb start
             EOF
+      - run:
+          name: Run tests
+          command: |
+            set -x
+            export ec2_ip=$(terraform -chdir=scripts/ci output -raw test_node_ssh)
+            files=$(ssh -o "StrictHostKeyChecking=no" ubuntu@$ec2_ip 'find /home/ubuntu/files/ -maxdepth 1 -mindepth 1')
+            for file in $files; do
+              ssh -o "StrictHostKeyChecking=no" ubuntu@$ec2_ip "sudo chmod +x $file; sudo $file"
```

Comment:
Instead of `chmod`-ing here, could we check in the test scripts as executables? The original bug report from the 1.8.7 packaging has gotten me nervous about not tracking things as executable in git

Reply:
Ah thanks, you've made me realize that not all the test scripts are checked in with +x, which they should have been.

However, I seemed to be having problems with the remote execution of these scripts without the explicit (redundant) chmodding, but maybe I was incorrect about the cause of those problems. I've removed the chmod pending successful tests.

Reply:
So yeah, the files did need to be explicitly  re-chmodded after being transferred to the host by terraform. I've moved this chmodding into the terraform config.